webknossos-datastore/app/com/scalableminds/webknossos/datastore/storage/Hdf5FileCache.scala

```diff
@@ -41,3 +48,26 @@ class Hdf5FileCache(val maxEntries: Int) extends LRUConcurrentCache[String, Cach
     }
   }
 }
+
+object CachedHdf5Utils {
+  def safeExecute[T](filePath: Path, meshFileCache: Hdf5FileCache)(block: CachedHdf5File => T)(
```

Comment:
Do you think the Box implementation could be used everywhere, and then be converted to Fox when needed? Or is there Fox-specific logic that is needed in some usages?